chef-utils/lib/chef-utils/dist.rb

```diff
@@ -15,6 +15,30 @@ class Automate
       PRODUCT = "Chef Automate"
     end
 
+    class Cli
+      # the chef-cli product name
+      PRODUCT = "Chef CLI"
+
+      # the chef-cli gem
+      GEM = "chef-cli"
+    end
+
+    class Compliance
+      # name of the compliance product
+      PRODUCT = "Chef Compliance"
```

Comment:
Do we really need this anymore now that compliance is EOL?

Reply:
Oh, didn't realize it was being EOL'd. Looks like it's being used here:

https://github.com/inspec/inspec/blob/66a372d71539b9132789aeb92b62690087ed34f2/lib/plugins/inspec-compliance/lib/inspec-compliance/api/login.rb#L26

FWIW I can remove it from this PR if it's going away eventually.

Reply:
The old standalone compliance server was never OSS so there's no need to dist that